docs/metrics.md

```diff
@@ -283,6 +274,7 @@ Returns the labels of the persistent volume claims that are used for restoring v
 Amount of active VNC connections, broken down by namespace and vmi name. Type: Gauge.
 
 ## Developing new metrics
-After developing new metrics or changing old ones, please run `make generate` to regenerate this document.
 
-If you feel that the new metric doesn't follow these rules, please change `doc-generator` with your needs.
+All metrics documented here are auto-generated and reflect exactly what is being
```

Comment:
@machadovilaca Why not keep the note about `make generate` ?

Reply:
Personal opinion, I like this wording better, and tbh if a developer didn't regenerate this document, he also did not read this section at the end of the file. We also already have the CI validating the update